src/__tests__/pages/news-announcements.test.tsx

```diff
@@ -73,11 +85,14 @@ describe('News page', () => {
         return Promise.resolve({ data: mockRssFeed })
       })
 
-      renderWithAuth(<News />)
+      renderWithAuth(<NewsAnnouncements announcements={cmsAnnouncementsMock} />)
+
+      // Slider component in react-slick clones each item in the carousel,
+      // so a length of 2 is accurate
+      expect(screen.getAllByText('Test Announcement')).toHaveLength(2)
```

Comment:
praise: thanks for including the note again here! This is something I'd find confusing without one :)